learning_journal/views/default.py

```diff
@@ -0,0 +1,67 @@
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPFound
+from ..models import Entry
+
+
+@view_config(
+    route_name='home', renderer='learning_journal:templates/home.html'
+)
+def home_view(request):
+    """Render home.html at '/'"""
+    journal_entries = request.dbsession.query(Entry).order_by(
+            Entry.creation_date.desc()
+    ).all()
+    return {
+        'title': 'Home',
+        'journal_entries': journal_entries
+    }
+
+
+@view_config(
+    route_name='new-entry',
+    renderer='learning_journal:templates/new-entry.html'
+)
+def new_entry(request):
+    """Render new-entry.html at '/new-entry'"""
+    title = body = error = ''
+    if request.method == 'POST':
+        title = request.params.get('title', '')
+        body = request.params.get('body', '')
+        if not title or not body:
+            error = 'Title and body are required'
+        else:
+            new_entry = Entry(title=title, body=body)
+            request.dbsession.add(new_entry)
+            return HTTPFound(location=request.route_url('home'))
```

Comment:
There are a couple of facts about this view that you should be able to test.  1.  That there is a new entry in the request.dbsession after it is run, 2. that the return value when it is run on POST is an HTTPFound exception object.  I don't see tests that cover that at all.